docs/recipes/precompiling-with-webpack.md

```diff
@@ -180,9 +181,11 @@ module.exports = {
 		rules: [
 			{
 				test: /\.(js|jsx)$/,
+        exclude: /(node_modules|bower_components)/,
 				use: {
 					loader: 'babel-loader',
```

Comment:
Does this need changing?

Reply:
It's not 'necessary' it was in the examples in babels docs. I can remove it for brevity.